test/test_socket_options.cpp

```diff
@@ -353,26 +457,46 @@ bool CheckInvalidValues(const OptionTestEntry& entry, SRTSOCKET sock, const char
     return true;
 }
 
-TEST_F(TestSocketOptions, DefaultVals)
+void TestDefaultValues(SRTSOCKET s)
 {
+    const char* test_desc = "[Caller, default]";
     for (const auto& entry : g_test_matrix_options)
     {
-        const char* test_desc = "[Caller, default]";
+        // Check flags. An option must be RW to test default value
+
+        if ( !(entry.flags & (Flags::R | Flags::W)) )
```

Comment:
An option does not have to be writable to tests its default value.

Reply:
There's no term of "default value" in case of options that can't be changed. Read-only options just read the value as the internals decide its value, and it can be different in different conditions. At best we can talk about the initial value of the option, that is, the value that the option has at the moment when the socket is fresh created and nothing is done with it yet.